packages/abi-typegen/src/abi/types/EvmAddressType.ts

```diff
@@ -7,7 +7,7 @@ export class EvmAddressType extends AType implements IType {
 
   public name = 'evmAddress';
 
-  static MATCH_REGEX: RegExp = /^struct EvmAddress$/m;
+  static MATCH_REGEX: RegExp = /^struct (std::vm::evm::evm_address::)?EvmAddress$/m;
```

Comment:
Once this becomes default behaviour, will we also be removing the optional regexes?

Reply:
Yes, we'll remove them for standard library types because they have a known specifier, but the generic ones like [enums](https://github.com/FuelLabs/fuels-ts/pull/2372/files/d3c8361493ed474f22861bcebba7fa0f1b4aac43#diff-4c608f8490ef0e210424f72efec83bec40a2a143be25af884a3d2029760c2152R18) and [structs](https://github.com/FuelLabs/fuels-ts/pull/2372/files/d3c8361493ed474f22861bcebba7fa0f1b4aac43#diff-6ec5eb9d704ba8a14d891a6981cbabf6cf5572aa9c7b5076529208d0f3512c4fR15) will have to remain because they can be imported from external libraries.